debian/changelog

```diff
@@ -1,11 +1,11 @@
-cloud-init (23.3-0ubuntu2) UNRELEASED; urgency=medium
+cloud-init (23.4~1g74e3e76b-0ubuntu1) UNRELEASED; urgency=medium
```

Comment:
Since we're no longer ubuntu/devel, this should be `23.3-0ubuntu0~23.10.1` now...but...it also doesn't really matter as we'll pull down the new version number when we release 23.4

Reply:
+1, I'll update it regardless